packages/docker/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM debian:jessie
+
+# add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
+RUN groupadd -r pelikan && useradd -r -g pelikan pelikan
+
+ENV PELIKAN_VERSION 0.1.2
+ENV PELIKAN_URL https://github.com/twitter/pelikan/archive/0.1.2.tar.gz
+ENV PELIKAN_SHA1 b8cb249fc44dc1a16f8ed9ce0be008c43ceb3599
+
+# build and install binaries, config
+RUN BUILDDEPS='cmake curl gcc libc6-dev make' \
+	&& set -x \
+	&& apt-get update && apt-get install --no-install-recommends -y $BUILDDEPS \
+	&& rm -rf /var/lib/apt/lists/* \
+	&& curl --insecure -SL "$PELIKAN_URL" -o pelikan.tar.gz \
```

Comment:
I assume you needed the `--insecure`, but it doesn't sound like a great solution. 😢